### PR TITLE
Create sensor.pl.json

### PR DESCRIPTION
**Pull request by nepozs**
Polish translation of sensor states created.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to an this integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example entry for `configuration.yaml`:

```yaml
# Example configuration.yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] The code has been formatted using Black (`black --fast custom_components`)

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated to README.md

### CODE REVIEW

**Comment by Limych** [comment]
And I never thought that Gismeteo was used in Poland! I thought he was only known in the CIS.

Thanks for the translation, but before I merge it, please make a small change that I have indicated below. This is done to ensure that users have a consistent picture in front of their eyes if they simultaneously use the weather object and sensors. The weather object uses the translation from here:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/weather/translations/pl.json

Your translation is better than that, but there is one place where the difference is too great.

You can add an edit simply by making a new commit to the branch from which you made this PR. The new commit will be added to the PR automatically.

File: custom_components/gismeteo/translations/sensor.pl.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "state": {
+    "gismeteo__condition": {
+      "clear-night": "Pogodna noc",
+      "cloudy": "Pochmurno",
+      "exceptional": "Wyjątkowa pogoda",
```
Comment (Limych) [inline custom_components/gismeteo/translations/sensor.pl.json]: Please, replace it to «Warunki nadzwyczajne» for consistency with weather object translation.

**Review by Limych — CHANGES_REQUESTED** [review]

File: custom_components/gismeteo/translations/sensor.pl.json (hunk 1 above)
Comment (nepozs) [reply custom_components/gismeteo/translations/sensor.pl.json]: Yes, "Warunki nadzwyczajne" should be used, I've edited patch-2 (but I'm still new to github, so I'm not sure if I did it properly).

I've found your "car wash" and "snowtire" integrations, they are so interesting, so I've checked Gismeteo service - it has also polish version https://www.gismeteo.pl/

Maybe your integrations will be more popular in Poland:
http://forum.jdtech.pl/Watek-hacs-home-assistant-community-store-alternatywny-sklep-z-dodatkami-dla-ha?pid=49523#pid49523
http://forum.jdtech.pl/Watek-hacs-home-assistant-community-store-alternatywny-sklep-z-dodatkami-dla-ha?pid=50069#pid50069
https://forum.arturhome.pl/t/prognoza-pogody-alternatywy-dla-preinstalowanego-met-no/2083
https://forum.arturhome.pl/t/integracje-okolopogodowe/2085

**Review by Limych — APPROVED** [review]
Thank you :heart:

**Pull request merged**